userguide/tutorials/plugin_development.adoc

```diff
@@ -189,6 +189,57 @@ When you start developing your own plugin, it would be useful to be able to set
 
 . This runs the application in debug mode. You can also set additional breakpoints as required.
 
+=== Authentication within plugins
+
+In order to invoke read/write API operations like `AccountUserApi#getAccountById` , plugin code must authenticate against Kill Bill first. Otherwise, it will result in an `org.apache.shiro.authz.UnauthenticatedException`. This section explains how authentication can be done.
```

Comment:
Actually, only write operations require authentication. `AccountUserApi#getAccountById` should work without having to log-in. Is your experience different?

Reply:
No sorry, it was incorrectly mentioned in the [notification plugin](https://docs.killbill.io/latest/notification_plugin.html#_how_to_develop_your_custom_notification_plugin) doc which I copied here. I have now fixed both.